examples-generated/networking/v1alpha1/loadbalancer.yaml

```diff
@@ -8,8 +8,8 @@ metadata:
   name: public
 spec:
   forProvider:
-    dropletIdsRefs:
-    - name: web
+    dropletIds:
+    - ${digitalocean_droplet.web.id}
```

Comment:
This doesn't seem right, no?

Reply:
Thats generated from the docs pages with upjet

Reply:
Is this because the API broke or because the docs itself changed?

It is a bit weird to suggest `dropletIds` over `dropletIdsRefs`